bke-development/gp-cicd-nexus/values.yaml

```diff
@@ -0,0 +1,30 @@
+nexus:
+  nexus:
+    docker:
+      enabled: true
+    resources:
+      requests:
+        cpu: "500m"
+        memory: "2048Mi"
+      limits:
+        cpu: "2000m" # According to UI 4 CPUs is recommended. Keep an eye on this.
+        memory: "3072Mi"
+  route:
+    enabled: true
+    name: nexus
+    path: "nexus.apps.$clustername.gepaplexx.com"
```

Comment:
Verständnisfrage für mich: Woher kommt das $clustername?

Reply:
nirgends. soll verdeutlichen, dass dieser Wert in der tatsächlichen ArgoCD Application überschrieben werden muss.